tabfilter.py

```diff
@@ -72,5 +82,13 @@ def make_tab(self, view):
 
 	def _on_done(self,index):
 		"""Callback handler to move focus to the selected tab index"""
+		if index == -1:
+			# If the selection was quit, re-focus the last selected Tab
+			self.window.focus_view(self.views[self.current_tab_idx])
+
 		if index > - 1:
```

Comment:
Now that we have two if conditions based on the `index`, can we turn this into an `if` and an `elif`, to avoid it having to evaluate the condition twice even if it's already passed the first check.  So end result will be something like:
```
if index == -1:
  ...
elif index > -1:
  ...
```